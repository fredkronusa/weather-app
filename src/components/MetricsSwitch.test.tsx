import * as React from "react";

import MetricsSwitchComponent from "./MetricsSwitch";
import { render } from "@testing-library/react";

const mockSetLocation = jest.fn();

describe("<MetricsSwitchComponent />", () => {
  // SNAPSHOT
  it("should match snapshot", async () => {
    const { asFragment } = render(
      <MetricsSwitchComponent
        setIsCelsius={() => mockSetLocation()}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
