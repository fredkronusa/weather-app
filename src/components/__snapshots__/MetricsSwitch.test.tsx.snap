// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MetricsSwitchComponent /> should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <input
      checked=""
      class="sc-AxirZ hXXLCO"
      id="celsius"
      name="unitType"
      type="radio"
      value="celsius"
    />
    <label
      class="sc-AxjAm fEPdEE"
      for="celsius"
    >
      ℃
    </label>
    <input
      class="sc-AxirZ hXXLCO"
      id="fahrenheit"
      name="unitType"
      type="radio"
      value="fahrenheit"
    />
    <label
      class="sc-AxjAm fEPdEE"
      for="fahrenheit"
    >
      ℉
    </label>
  </div>
</DocumentFragment>
`;
